main.go

```diff
@@ -1412,7 +1443,7 @@ type pkgCache struct {
 	//
 	// This record is necessary for knowing what names from imported packages
 	// weren't obfuscated, so we can obfuscate their local uses accordingly.
-	ReflectObjects map[objectString]struct{}
+	ReflectObjects map[objectString]string
```

Comment:
update the godoc with what you're storing as the map value now, which I assume to be the original object name. I'd rename the map too, e.g. `ReflectObjectNames`.